### ISSUE TITLE
Bugs on the UI

### ISSUE THREAD

**Issue by vadimnikonov1** [zkBob/zkbob-ui#104]
1. On the start page when we use the app for the first time, replace GET STARTED with CONNECT WALLET. 
Or leave the GET STARTED button, but split it into two actions: CREATE ZK ACCOUNT and CONNECT ZK ACCOUNT. 
Or make two separate buttons CREATE ZK ACCOUNT and CONNECT ZK ACCOUNT, eliminating unnecessary transition from the GET STARTED button.

I propose to divide the actions as follows:

CREATE ZK ACCOUNT:
Create a new zkBob Account with
MetaMask (a wallet is connected)
WalletConnect (a wallet is connected)
Secret phrase (a new phrase is created)

CONNECT ZK ACCOUNT:
Connect zkBob Account with
MetaMask (this button connects zk if MM is already connected or redirects to connect MM)
WalletConnect (this button connects zk if WC is already connected or redirects to WC connection)
Secret phrase (user enters the already created secret phrase)

2. DEPOSIT:
When a deposit is sent, a different result is indicated: 
Above Deposit sent - below Deposit is in process.
Deposit sent - must be written in passive voice as the deposit hasn't sent itself but has been sent by someone, i.e. two options: Deposit was sent or Deposit has been sent.

![Image](https://user-images.githubusercontent.com/84346846/204161170-aba28c86-1dbc-47c9-95d2-e0c59ca50e02.png)

When we open the pop-up "Deposit sent" and quickly press the Got it! button, the user is notified that his transaction is not completed and the page cannot be reloaded for at least 30 seconds. But if the user decides to read all the information in that window, he will not receive this notification. 
Suggestion: first write Deposit is in process (the Got it!  button is inactive or not present at all, as the transaction is not completed), then Deposit has been sent (the Got it! button is active).

3. TRANSFER and MULTITRANSFER:
Enter address - we need to add THE. We ask for the specific address of the recipient.
When Multitransfer is selected, the Amount of BOB is written at the top. Amount - written with a capital letter after a comma. Is it ok?

![Image](https://user-images.githubusercontent.com/84346846/204161510-7bfbd2ad-68ba-446a-b1b5-99520a6a67c3.png)

1 address per line - but the address is written in two lines. 

![Image](https://user-images.githubusercontent.com/84346846/204162258-3fb67917-e216-4c69-86c9-5001c36df9fa.png)

If you put a comma on both sides of the number of BOB, you get an error - 1 rows with incorrect addresses or formatting issues. 
If 1, we can not use Rows, only Row. Line and Row are not the same thing? Wouldn't it be better to write the same in two places? For example: 1 address per LINE and 1 LINE with incorrect addresses or formatting issues.
THE Sender, recipient and amount are never disclosed.

![Image](https://user-images.githubusercontent.com/84346846/204163239-a85f0985-c594-493a-be8a-543ad998d45b.png)

Duplicate addresses found - Duplicate addresses ARE found.

![Image](https://user-images.githubusercontent.com/84346846/204163254-f9ad215a-86a0-410d-bab0-16608009f124.png)

will be transferred to 1 zkBob addresses - if to 1 address, remove the plural and leave address. A multitransfer involves sending to multiple addresses, so it's not correct to allow one address. I suggest the following: if a user has entered one address, the Proceed button should be disabled and the prompt "Enter two or more addresses" should be displayed.

![Image](https://user-images.githubusercontent.com/84346846/204162291-123e3b3c-92e7-47e5-a475-6117aaec5398.png)

Latest Transfer, Latest Withdrawal, View History - should the second word be capitalised?

![Image](https://user-images.githubusercontent.com/84346846/204162205-8ef36791-3238-4fc3-8755-4d06c26999c1.png)

4. OTHERS
You can deposit, transfer or withdraw funds once the transaction is complete COMPLETED (passive voice in Present Simple, it doesn't complete itself, it is done with someone else's help). 
Do not refresh the page for at least 30 seconds! Transaction status will update automatically. Swap the top and bottom sentences, the user won't have time to read that the page cannot be refreshed.

Please wait for your transaction to finalize - WILL BE FINALIZED.

![Image](https://user-images.githubusercontent.com/84346846/204163357-cc89f7d9-1375-4c65-9c8d-3d7ec65d7317.png)

To enhance security, THE password entry is required each time a page IS reloads REFRESHED.

![Image](https://user-images.githubusercontent.com/84346846/204163378-007cfadd-a011-4e61-aee9-d2a5687b90ad.png)

Replace SECRET RECOVERY PHRASE with SEED PHRASE, as this phrase doesn't restore anything at the Create account stage, it is still needed for registration/creation of an account.

When we create an account with Secret Phrase, when selecting words for a phrase, there is no option to delete the last word if the user selected it incorrectly. 

![Image](https://user-images.githubusercontent.com/84346846/204162310-66564bde-c9b3-4e9c-8130-fa139d1ebf84.png)

Enter AN amount. The amount has already been determined by the user, so we need to put THE or no article in front of the amount.

![Image](https://user-images.githubusercontent.com/84346846/204163440-4f23d3ed-5941-4789-9c0f-061532b24492.png)
